specs/sequencer-relayer.md

```diff
@@ -0,0 +1,152 @@
+# Sequencer-Relayer
+
+## Overview
+
+The sequencer-relayer, a.k.a relayer, is an application whose primary
+responsibility is to store sequenced rollup data on a remote data availability
+layer (DA layer).
+
+The immediate source of the sequenced data is the sequencer blockchain network,
+where rollups' data is sequenced and included in sequencer blocks, while the DA
+layer is the Celestia blockchain.
+
+## Application Logic
+
+The application logic of the relayer is comprised of three main tasks: getting
+the data from a sequencer node, transforming the data in preparation for
+storing, and putting the transformed data onto the Celestia network.
+
+### Getting the Data from a Sequencer Node
+
+The relayer runs an endlessly repeating task, the "reader task", which does the
+following:
+
+1. polls the sequencer node for the latest (highest) sequencer block height at
+  a fixed frequency (once per second currently)
+1. upon learning of a new height, fetches the block at that height from the
+  sequencer node
+1. upon receiving the sequencer block, forwards it to the transformation task
```

Comment:
```suggestion
2. upon learning of a new height, fetches the block at that height from the
  sequencer node
3. upon receiving the sequencer block, forwards it to the transformation task
```

Reply:
The numbers are rendered correctly in MarkDown and I tend to use all 1s since it's easier to keep the numbered list correct if you reorder, add or remove items.

Happy to change it if you prefer though - not a big deal :)

Reply:
oh interesting, i didn't read the rendered markdown lol, no worries then